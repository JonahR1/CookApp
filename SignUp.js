import React from 'react';
import { StyleSheet , View, TextInput, Button} from 'react-native'

/*class SignUp extends React.Component{
    constructor(props){
        super(propos)

    }
    }
}*/