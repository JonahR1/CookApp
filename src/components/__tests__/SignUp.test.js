import React from 'react';
import SignUp from '../SignUp';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(
    <SignUp />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});