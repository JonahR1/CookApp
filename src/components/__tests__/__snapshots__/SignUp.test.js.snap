// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Text
    style={
      Object {
        "color": "black",
        "fontSize": 23,
        "fontWeight": "bold",
      }
    }
  >
    Email
  </Text>
  <TextInput
    allowFontScaling={true}
    placeholder="Enter your Email"
    rejectResponderTermination={true}
    style={
      Object {
        "color": "white",
        "fontSize": 23,
        "marginVertical": 10,
        "width": "100%",
      }
    }
    underlineColorAndroid="transparent"
  />
  <Text
    style={
      Object {
        "color": "black",
        "fontSize": 23,
        "fontWeight": "bold",
      }
    }
  >
    Username
  </Text>
  <TextInput
    allowFontScaling={true}
    placeholder="Enter your Username"
    rejectResponderTermination={true}
    style={
      Object {
        "color": "white",
        "fontSize": 23,
        "marginVertical": 10,
        "width": "100%",
      }
    }
    underlineColorAndroid="transparent"
  />
  <Text
    style={
      Object {
        "color": "black",
        "fontSize": 23,
        "fontWeight": "bold",
      }
    }
  >
    Password
  </Text>
  <TextInput
    allowFontScaling={true}
    placeholder="Enter your Password"
    rejectResponderTermination={true}
    style={
      Object {
        "color": "white",
        "fontSize": 23,
        "marginVertical": 10,
        "width": "100%",
      }
    }
    underlineColorAndroid="transparent"
  />
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "purple",
        "borderRadius": 5,
        "color": "white",
        "fontSize": 33,
        "fontWeight": "bold",
        "opacity": 1,
        "textAlign": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "backgroundColor": "purple",
          "borderRadius": 5,
          "color": "white",
          "fontSize": 33,
          "fontWeight": "bold",
          "textAlign": "center",
        }
      }
    >
      Submit
    </Text>
  </View>
</View>
`;
